File: FrontEnd/Projects/Blazor/Transfers/Entities/TransferEntity.cs
using Blazor.Estates.Entities;
using Blazor.Users.Entities;

namespace Blazor.Transfers.Entities;

public class TransferEntity
{
    public UserEntity notary { get; set; }
    public UserEntity buyer { get; set; }
    public EstateEntity estate { get; set; }
    public DateTime date_created { get; set; }
    public bool has_seller_signed { get; set; }
    public bool has_buyer_signed {get; set; }
    public bool is_finalized { get; set; }
    public DateTime date_finalized { get; set; }
    public bool is_payed { get; set; }
    public bool is_posted { get; set; }
}

File: Application/src/main/java/com/dfm/Transfers/Entities/TransferEntity.java
package com.dfm.Transfers.Entities;

import java.util.Date;

import com.dfm.Estates.Entities.EstateEntity;
import com.dfm.Users.Entities.UserEntity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
public class TransferEntity
{
    private UserEntity notary;
    private UserEntity buyer;
    private EstateEntity estate;
    private Date date_created;
    private boolean has_seller_signed;
    private boolean has_buyer_signed;
    private boolean is_finalized;
    private Date date_finalized;
    private boolean is_payed;
    private boolean is_posted;
}
